taxi/views.py

```diff
@@ -1,9 +1,12 @@
+from gc import get_objects
```

Comment:
The import `get_objects` from the `gc` module is not used in this file. Consider removing it to clean up the code.